src/views/preview/preview.jsx

```diff
@@ -41,6 +44,7 @@ class Preview extends React.Component {
         // parts[2]: undefined if no :id, otherwise either 'editor' or 'fullscreen'
         this.state = {
             editable: false,
+            extensions: [],
             favoriteCount: 0,
             loveCount: 0,
             projectId: parts[1] === 'editor' ? 0 : parts[1],
```

Comment:
We already possibly have a projectId, so we should kick off getExtensions with this initial state.

Reply:
Was this meant to be a comment on line 80? Not sure initializing the extension list to anything but an empty array makes sense.

Reply:
Ah gotcha, that makes sense.